src/cmakeProject.ts

```diff
@@ -1963,7 +2156,29 @@ export class CMakeProject {
 
     async ctest(): Promise<number> {
         const drv = await this.preTest();
-        return (await this.cTestController.runCTest(drv)) ? 0 : -1;
+        const retc = await this.cTestController.runCTest(drv);
```

Comment:
Why the change to check for if `retc` is defined? The return type of `runCTest` is a `number`, so it should never return `undefined`.

Reply:
We should just be returning retc.

Reply:
For cpack I did the same as for ctest. If thinking about both, I thought the number return codes greater than 0 mean success and those we would map to a 0 return code, whether any <=0 return code we'd want to be -1. I did not see it as a query for undefined but a query for positive value number. I can't seem to find documentation spec about ctest and cpack return codes.

Reply:
Ah, I see now. That makes sense. Better to leave it as is. THanks!